### PR TITLE
Min/max temperature for forecast objects

### PR DESCRIPTION
**Pull request by RonnyAL**
A first attempt at implementing this. Relies on some unfortunate changes (requires `max_hour=24` instead of the original 6 and that the hour of the datetime passed to `get_weather` from `get_forecast` is 0), but hopefully this is something we can work with!

### CODE REVIEW

File: metno/__init__.py [hunk 1]
```diff
@@ -101,12 +101,12 @@ def get_forecast(self, time_zone):
         if self.data is None:
             return []
 
-        now = datetime.datetime.now(time_zone).replace(hour=12, minute=0,
+        now = datetime.datetime.now(time_zone).replace(hour=0, minute=0,
```
Comment (Danielhiversen) [inline metno/__init__.py]: Then the forecast will be for midnight, and not for 12:00.

File: metno/__init__.py [hunk 2]
```diff
@@ -101,12 +101,12 @@ def get_forecast(self, time_zone):
...
                                                        second=0, microsecond=0)
         times = [now + datetime.timedelta(days=k) for k in range(1, 6)]
         return [self.get_weather(_time) for _time in times]
 
-    def get_weather(self, time, max_hour=6):
+    def get_weather(self, time, max_hour=24):
```
Comment (Danielhiversen) [inline metno/__init__.py]: Then the displayed forecast might not correspond to the displayed date

File: metno/__init__.py [hunk 3]
```diff
@@ -138,8 +138,25 @@ def get_weather(self, time, max_hour=6):
         res['humidity'] = get_data('humidity', ordered_entries)
         res['wind_speed'] = get_data('windSpeed', ordered_entries)
         res['wind_bearing'] = get_data('windDirection', ordered_entries)
+        minmax = get_min_max(ordered_entries)
+        res['templow'] = minmax['min']
```
Comment (Danielhiversen) [inline metno/__init__.py]: Why do you have to search for min temperature?
Is that not given in the data from met.no?

File: metno/__init__.py [hunk 4]
```diff
@@ -138,8 +138,25 @@ def get_weather(self, time, max_hour=6):
...
+        res['temphigh'] = minmax['max']
```
Comment (Danielhiversen) [inline metno/__init__.py]: Is the max temp missing from the met.no data?

File: metno/__init__.py (hunk 3 above)
Comment (RonnyAL) [reply metno/__init__.py]: I found attributes "minTemperature" and "maxTemperature" in the API response, but to me it looked like these indicate that they predict a temperature for some particular timespan that falls within this range (min-max). So as far as I can see, there's no "minTemperature" or "maxTemperature" on a per-day basis.

File: metno/__init__.py (hunk 1 above)
Comment (RonnyAL) [reply metno/__init__.py]: Yeah, that's a problem. To be honest I haven't completely understood yet how the forecast for a particular day is given. Is the idea maybe that if `max_hour` was set to 6, and hour (here) to 12, the available forecast closest to (but no more than 6 hours away from) hour 12 would be returned?

**Pull request closed without merging**